### PR TITLE
Azimuth.exe

### PR DESCRIPTION
**Pull request by memori034**
need pid fixes

### CODE REVIEW

**Review by realjoshuau — CHANGES_REQUESTED** [review]
do you need the azimuth direction to switch in between matches? you can get it from the DriverStation class

**Comment by realjoshuau** [comment]
ALSO LINT YO STUFF

**Comment by memori034** [comment]
> do you need the azimuth direction to switch in between matches? you can get it from the DriverStation class

ok it now changes with ds alliance

**Comment by memori034** [comment]
> ALSO LINT YO STUFF

gradle skill issues im not tryna fix

**Comment by memori034** [comment]
we test after avr

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -274,29 +277,123 @@ private void configureClimb() {
...
+        new InstantCommand(
+            () -> {
+              if (DriverStation.getAlliance().toString().indexOf('R') != -1) {
+                isRed = true;
+              } else {
+                isRed = false;
+              }
+            });
+
+    if (isRed = true) { // RED
```
[the remark this answers is not in the document]
Comment (memori034) [reply src/main/java/frc/robot/RobotContainer.java:311]: meow:3

[the remark this answers is not in the document]
Comment (memori034) [reply src/main/java/frc/robot/RobotContainer.java:311]: i fix

**Pull request merged**